files/fr/web/html/link_types/modulepreload/index.html

```diff
@@ -0,0 +1,15 @@
+---
+title: 'Link types: modulepreload'
+slug: Web/HTML/Link_types/modulepreload
+browser-compat: html.elements.link.rel.modulepreload
+translation_of: 'Web/HTML/Link_types/modulepreload'
+---
+<p><span class="seoSummary">Le mot-clé <strong><code>modulepreload</code></strong> de l'attribut {{HTMLAttrxRef("rel", "link")}} de l'élément {{HTMLElement("link")}} fournit une manière déclarative d'analyser de façon préventive un <a href="/fr/docs/Web/JavaScript/Guide/Modules">module JavaScript</a> et ses dépendances, ainsi que de les stocker dans la cartes des modules du document pour une évaluation future.</span></p>
```

Comment:
cf. https://github.com/mdn/translated-content/pull/1794/files#r679360769

and also a minor typo "la carteS" > "la carte"

Reply:
Yep, fixed, thanks! https://github.com/mdn/translated-content/pull/1796/commits/a1a681e59258be0687f32b8b6605152524f83d6c